social_distribution/sd/templates/sd/index.html

```diff
@@ -12,34 +12,57 @@
 	<link rel="stylesheet" href="{% static '../../static/sd/style.css' %}">
 </head>
 
+<!-- function to show comments
+Reference: https://stackoverflow.com/questions/29321494/show-input-field-only-if-a-specific-option-is-selected/29321711
+Author: https://stackoverflow.com/users/4721273/josephus87--> 
+<script type="text/javascript">
+function displayCommentsCheck(that) {
+    if (document.getElementById("showComments").style.display == "block") {
+        document.getElementById("showComments").style.display = "none";
+    } else {
+        document.getElementById("showComments").style.display = "block";
+    }
+}
+
+function displayAddComments(that) {
+    if (document.getElementById("addComment").style.display == "block") {
+        document.getElementById("addComment").style.display = "none";
+    } else {
+        document.getElementById("addComment").style.display = "block";
+    }
+}
+</script>
+
 <body>
```

Comment:
Do we want to throw this into a separate js file by any chance?

Reply:
(We're going to have to use this code again in every page that has posts)

Reply:
Agreed, let's try and keep high level code separate to avoid duplicated work and inconsistencies